Lesson 6 task 01

gemansipator opened the pull request:
Программа вводит строки, пока пользователь не введёт пустую строку (нажав enter).
 Потом она конвертирует строки в верхний регистр (Мама превращается в МАМА) и выводит их на экран.


 Новая задача: Программа вводит строки, пока пользователь не введёт пустую строку (нажав enter).
 Если в строке чётное число букв, строка удваивается, если нечётное - утраивается.
 Программа выводит слова на экран.

gemansipator committed:
Task01

gemansipator committed:
Task01 Программа вводит строки, пока пользователь не введёт пустую строку (нажав enter).
        Потом она конвертирует строки в верхний регистр (Мама превращается в МАМА) и выводит их на экран.

        Новая задача: Программа вводит строки, пока пользователь не введёт пустую строку (нажав enter).
        Если в строке чётное число букв, строка удваивается, если нечётное - утраивается.
        Программа выводит слова на экран.

RinatBeybutov commented on src/main/java/site/javadev/lesson_6/homework/Task01.java:
```diff
@@ -0,0 +1,47 @@
+            String upperCaseLetters = input.toUpperCase();
+
+            if (upperCaseLetters.length() % 2 == 0) {
+                upperCaseLetters = upperCaseLetters + upperCaseLetters;
```
здесь использовать String.repeat

gemansipator replied:
Ок

RinatBeybutov commented on src/main/java/site/javadev/lesson_6/homework/Task01.java:
```diff
@@ -0,0 +1,47 @@
+                upperCaseLetters = upperCaseLetters + upperCaseLetters;
+                System.out.println( upperCaseLetters);
+            }else {
+                upperCaseLetters = upperCaseLetters + upperCaseLetters +upperCaseLetters;
```
здесь использовать String.repeat

gemansipator committed:
Task01 Программа вводит строки, пока пользователь не введёт пустую строку (нажав enter).
        Потом она конвертирует строки в верхний регистр (Мама превращается в МАМА) и выводит их на экран.

        Новая задача: Программа вводит строки, пока пользователь не введёт пустую строку (нажав enter).
        Если в строке чётное число букв, строка удваивается, если нечётное - утраивается.
        Программа выводит слова на экран.

RinatBeybutov commented:
принято
